packages/aws-cdk/lib/cli.ts

```diff
@@ -92,6 +92,8 @@ async function parseCommandLineArguments() {
     .command('bootstrap [ENVIRONMENTS..]', 'Deploys the CDK toolkit stack into an AWS environment', (yargs: Argv) => yargs
       .option('bootstrap-bucket-name', { type: 'string', alias: ['b', 'toolkit-bucket-name'], desc: 'The name of the CDK toolkit bucket; bucket will be created and must not exist', default: undefined })
       .option('bootstrap-kms-key-id', { type: 'string', desc: 'AWS KMS master key ID used for the SSE-KMS encryption', default: undefined, conflicts: 'bootstrap-customer-key' })
+      .option('use-default-permissions-boundary', { type: 'boolean', alias: ['dpb', 'default-permissions-boundary'], desc: 'Use the default permissions boundary.', default: undefined, conflicts: 'permissions-boundary' })
```

Comment:
```suggestion
      .option('create-example-permissions-boundary', { type: 'boolean', alias: ['epb', 'example-permissions-boundary'], desc: 'Use the example permissions boundary.', default: undefined, conflicts: 'permissions-boundary' })
```

What about this name instead?

Reply:
I'm thinking of the middle option `use-example-permissions-boundary` since the cli param does not always create one.